reactapp/src/components/ListElement/ListElement.js

```diff
@@ -0,0 +1,14 @@
+export const ListElement = ({children}) => {
+
+  const handleClick = () => {
```

Comment:
musisz przekazac to children jako parametr funkcji, bo jak tego nie zrobisz to console log nie wie co ma wyswietlic

const handleClick = ( children ) => { ...